.github/workflows/testing.yml

```diff
@@ -73,15 +72,21 @@ jobs:
           python -m pip install --upgrade -r dev-requirements.txt
           python -m pip install --upgrade .
       - name: Try single CLI run of tool
+        env:
+          nvd_api_key: ${{ secrets.NVD_API_KEY }}
         run: |
           NO_EXIT_CVE_NUM=1 python -m cve_bin_tool.cli test/assets/test-kerberos-5-1.15.1.out -n json
       - name: Run async tests
+        env:
+          nvd_api_key: ${{ secrets.NVD_API_KEY }}
```

Comment:
Hopefully not needed if we've done our caching and moved tests to longtests, but I wouldn't swear we're done with those so let's keep it.